internal/npm_install/generate_build_file.js

```diff
@@ -1024,6 +1042,26 @@ nodejs_binary(
   return result;
 }
 
+function printIndexBzl(pkg) {
+  let result = '';
+  const executables = _findExecutables(pkg);
+  if (executables.size) {
+    result = `load("@build_bazel_rules_nodejs//:defs.bzl", "npm_package_bin")
+
+`;
+  }
+  for (const name of executables.keys()) {
+    result = `${result}
+
+# Generated helper macro to call ${name}
+def ${name}(**kwargs):
+    npm_package_bin(package = "${pkg._dir}", package_bin = "${name}", **kwargs)
```

Comment:
I think specific `tool` here instead of `package` and `package_bin` as the `@npm` workspace may not be `@npm` for this npm_install/yarn_install

Reply:
yes good call